db/schema.rb

```diff
@@ -10,143 +10,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_01_10_143925) do
+ActiveRecord::Schema.define(version: 2019_01_18_093814) do
 
-  create_table "cpi_fixed_data", force: :cascade do |t|
+  create_table "fixed_data", force: :cascade do |t|
     t.string "year"
-    t.decimal "cpi"
-    t.decimal "real"
-    t.decimal "fr_uplift"
-    t.decimal "rc_factor"
     t.decimal "rc_uplift"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
-  create_table "removals", force: :cascade do |t|
+  create_table "rent_charges", force: :cascade do |t|
     t.string "uprn"
     t.string "address"
     t.string "address_2"
     t.string "comments"
     t.decimal "rr_count"
-    t.string "property_type"
-    t.decimal "base_data_bed_size"
+    t.integer "property_type"
+    t.integer "base_data_bed_size"
     t.decimal "bedroom_weight"
     t.string "mra_archetype"
     t.decimal "jan_1999_asset_values"
     t.decimal "formula_rent_prev_year"
```

Comment:
do we need `formula_rent_prev_year` and `rent_cap_prev_year` now that the table will contain rent details for all years? Also, I know a lot of the rent details can be calculated dynamically but don't we need to persist at least the current rent charge to the db? To me it makes sense to persist `formula_rent_this_year`, `rent_cap_this_year`, and  `uprated_actual`, and everything else can be calculated dynamically.

Reply:
Agree with this 100%